### PR TITLE
Move documentation contributing.md and add Governance.md and Contributing.md

### PR DESCRIPTION
**Pull request by yhuiYH**
- Moved documentation contributing.md to net path under /docs/contribute/contributing.md
- Adding Governance.md and Contributing.md for review

### CODE REVIEW

File: docs/contribute/contribute-docs.md
Comment (saadrahim) [inline docs/contribute/contribute-docs.md:1]: This file should be moved using git mv rather than recreated.

File: CONTRIBUTING.md
Comment (saadrahim) [inline CONTRIBUTING.md:92]: we are github based for now and as far as we can see in the future. This section appears redundant.

File: GOVERNANCE.md
Comment (saadrahim) [inline GOVERNANCE.md:20]: legal name of AMD?

File: GOVERNANCE.md
Comment (saadrahim) [inline GOVERNANCE.md:33]:
```suggestion
repositories. they must use pull requests (PRs) for all changes.
```
Not sure what this sentence means.

File: GOVERNANCE.md
Comment (saadrahim) [inline GOVERNANCE.md:48]:
```suggestion
  [the rocm repository](https://github.com/RadeonOpenCompute/ROCm) and other repositories. AMD has adopted docs-as-code for documentation. Documentation is uses the same source code management(SCM) system, git, as the rest of our source code.
```

File: docs/contribute/contribute-docs.md
Comment (saadrahim) [inline docs/contribute/contribute-docs.md:3]: We can reduce the focus of this doc as the new contribution guide covers the first paragraph.

**Review by saadrahim — CHANGES_REQUESTED** [review]

File: docs/contribute/contribute-docs.md
Comment (LisaDelaney) [reply docs/contribute/contribute-docs.md:1]: There are now two different contributing files, so although it looks like it was recreated, it's the same as what we had and 'contributing' now has totally different content.

File: docs/contribute/contribute-docs.md
Comment (LisaDelaney) [reply docs/contribute/contribute-docs.md:3]: it's really just the first sentence and I think it's an okay way to lead into the content...

File: docs/contribute/contribute-docs.md
Comment (yhuiYH) [reply docs/contribute/contribute-docs.md:1]: It looks like history is retained (I think that's the motivation behind using "git mv")?  
https://github.com/RadeonOpenCompute/ROCm/commits/amd/yhui/governance_and_contributing/docs/contribute/contribute-docs.md

File: CONTRIBUTING.md
Comment (yhuiYH) [reply CONTRIBUTING.md:92]: There was suggestion for the document the be GitHub agnostic.  Or at least highlight that we're currently using GitHub, but that can change, right, to cover bases.  I added these sections to address those concerns.

"I think there was general consensus around making any infrastructure requirements vendor agnostic in case we decide to switch at some point.  That said, the document could be construed as describing our current infrastructure process and some of the language could be made more generic and we could add a section explicitly stating that the current model uses GitHub, but that similar processes would be applicable if that infrastructure ever changed."

File: docs/contribute/contribute-docs.md
Comment (yhuiYH) [reply docs/contribute/contribute-docs.md:3]: Right, I liked your first sentence so used it in the new Contributing.md.  It's a very welcoming opener. I think you can keep it here too.

File: GOVERNANCE.md
Comment (LisaDelaney) [reply GOVERNANCE.md]: I don't have super strong feelings (tend to prefer 'AMD')...is there a reason (legal or otherwise) for having the long name?

Comment (yhuiYH) [reply GOVERNANCE.md:20]: Sorry, I reverted this back to just AMD.  It sounds better to me.

**Comment by yhuiYH** [comment]
Ben Sanders approved the Upstream Workgroup docs including this GOVERNANCE.md and CONTRIBUTING.md at 12/15 meeting so contents is approved.  Lisa reviewed from tech writer POV, so I think this, pending approval by code-owners, is ready to push out to GitHub.

**Review by saadrahim — APPROVED** [review]

**Pull request merged**
